packages/pysam/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: pysam
+  version: 0.22.0
+  top-level:
+    - pysam
+source:
+  url: https://files.pythonhosted.org/packages/35/22/3d01778c13f1103401313f1232c1c0596d97aaee21c1d60564640f3049bd/pysam-0.22.0.tar.gz
+  sha256: ab7a46973cf0ab8c6ac327f4c3fb67698d7ccbeef8631a716898c6ba01ef3e45
+  patches:
+    - patches/0001-remove_config_library.patch
+    - patches/0001-use-emmake.patch
+    - patches/0001-remove-darwin-check.patch
+requirements:
+  host:
+    - zlib
+build:
+  cflags: |
+    -I$(WASM_LIBRARY_DIR)/include
+    -fPIC
+  cross-script: |
```

Comment:
I guess these commands inside `cross-script` are for debugging. Could you delete them?